### PR TITLE
Vaurca Bite Nerf

### PR DESCRIPTION
**Pull request by Geevies**
* Vaurca warriors now need a choking grip to mandible bite someone. Doing the attack drops the grab, preventing a headbutt. The hard cooldown has increased to five seconds.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Geevies** [comment]
would it be too on the nose to call it a _bug_ fix, eh, eh?

**Comment by Leudoberct1** [comment]
I disagree with the requirement for a red grip, this makes the bite effectively useless on anyone but people who are already prone, I would prefer to keep it at blue grip- this makes it more viable for fighting carp as well, which is what I often use it for. As for the dropping of the grab and the increase on the cooldown, I am in favour.

**Comment by runecap** [comment]
Agree with leudo, making it require stranglegrab makes it useless- especially since you then LOSE that grab and stranglegrabs take a while to get- meaning it would be better to literally just slash them.

**Comment by Geevies** [comment]
The bite can instantly crack bones, I don't think that's an ability that should be pulled off in the fly.

**Comment by Leudoberct1** [comment]
I have not once ever seen it fracture someone who was already uninjured with a single bite. It has always taken two bites minimum.

**Comment by runecap** [comment]
It will always take two bites because bone-cracks are at 30 brute and bites only do 25. It should be noted that it's pretty easy to resist out of a grab (which usually leads to that person keeping their distance from you afterwards)

**Comment by Doxxmedearly** [comment]
Add the full mask/helmet check you discovered was missing and it should be fine as a blue grip.

**Comment by Leudoberct1** [comment]
Still in favour of dropping the grab down to blue.

**Comment by skull132** [comment]
A strong attack must require a commitment. 25 damage is a strong attack, 2nd stage grip is fuck-all for commitment. This will effectively religate it to being a punishing move or a finisher, yes, but such is the role of strong attacks.
A weak attack can be quick. But 25 damage is way too much for a weak attack.

**Comment by runecap** [comment]
Before, I thought it was neckgrab-only....then I learned you can do it with bluegrab. At this point, won't a switch to just having it need to be neckgrab while getting rid of biting dropping the grab be best? If you drop the grab, it means choking out/throat-slitting is a better option because those don't get rid of the grab and still do damage.

File: code/modules/mob/living/carbon/human/human_powers.dm [hunk 1]
```diff
@@ -337,43 +337,51 @@ mob/living/carbon/human/proc/change_monitor()
...
 		return
 
-	if(G.state < GRAB_AGGRESSIVE)
-		to_chat(src, "<span class='warning'>You must have an aggressive grab to gut your prey!</span>")
+	if(G.state < GRAB_KILL)
+		to_chat(src, SPAN_WARNING("You must have an strangling grip to bite someone!"))
 		return
 
-	if(istype(G.affecting,/mob/living/carbon/human))
+	if(istype(G.affecting, /mob/living/carbon/human))
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/human_powers.dm]:
```suggestion
	if(ishuman(G.affecting))
```

File: code/modules/mob/living/carbon/human/human_powers.dm [hunk 2]
```diff
@@ -337,43 +337,51 @@ mob/living/carbon/human/proc/change_monitor()
...
 			return
-		M.apply_damage(25,BRUTE, sharp=1, edge=1)
-		visible_message("<span class='warning'><b>\The [src]</b> rips viciously at \the [G.affecting]'s flesh with its mandibles!</span>")
+		M.apply_damage(25, BRUTE, sharp = TRUE, edge = TRUE)
+		visible_message(SPAN_WARNING("<b>\The [src]</b> rips viciously at \the [G.affecting]'s flesh with its mandibles!"))
 		msg_admin_attack("[key_name_admin(src)] mandible'd [key_name_admin(M)] (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[src.x];Y=[src.y];Z=[src.z]'>JMP</a>)",ckey=key_name(src),ckey_target=key_name(M))
-	playsound(src.loc, 'sound/weapons/slash.ogg', 50, 1)
-	last_special = world.time + 25
+	qdel(G)
+	playsound(get_turf(src), 'sound/weapons/slash.ogg', 50, TRUE)
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/human_powers.dm]: Isn't there a bite sound for this?

Comment (Geevies) [reply code/modules/mob/living/carbon/human/human_powers.dm]: Seems a bit out of scope to me

File: html/changelogs/geeves-vaurca_grab_bite_nerf.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - tweak: "Vaurca warriors now need a neck grip to mandible bite someone. Doing the attack drops the grab, preventing a headbutt. The hard cooldown has increased to five seconds."
```
Comment (NonQueueingMatt) [inline html/changelogs/geeves-vaurca_grab_bite_nerf.yml]:
```suggestion
  - tweak: "Vaurca warriors now need a neck grip to mandible bite someone. The hard cooldown has increased to five seconds."
```

File: code/modules/mob/living/carbon/human/human_powers.dm [hunk 4]
```diff
@@ -337,43 +337,51 @@ mob/living/carbon/human/proc/change_monitor()
...
 		if(!istype(M))
 			return
-		M.apply_damage(25,BRUTE, sharp=1, edge=1)
-		visible_message("<span class='warning'><b>\The [src]</b> rips viciously at \the [G.affecting]'s flesh with its mandibles!</span>")
+		M.apply_damage(25, BRUTE, sharp = TRUE, edge = TRUE)
+		visible_message(SPAN_WARNING("<b>\The [src]</b> rips viciously at \the [G.affecting]'s flesh with its mandibles!"))
 		msg_admin_attack("[key_name_admin(src)] mandible'd [key_name_admin(M)] (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[src.x];Y=[src.y];Z=[src.z]'>JMP</a>)",ckey=key_name(src),ckey_target=key_name(M))
-	playsound(src.loc, 'sound/weapons/slash.ogg', 50, 1)
-	last_special = world.time + 25
+	qdel(G)
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/human_powers.dm]: you forgot to remove this

Comment (Geevies) [reply code/modules/mob/living/carbon/human/human_powers.dm]: i'm keeping the grab drop.

File: html/changelogs/geeves-vaurca_grab_bite_nerf.yml (hunk 3 above)
Comment (Geevies) [reply html/changelogs/geeves-vaurca_grab_bite_nerf.yml]: No.

**Comment by runecap** [comment]
Having it be neckgrab and having it then drop that is big enough, but needing a choke and then having it drop the grab means you're better off using almost any other grab-maneuver than the ability.  This is too harsh.

**Comment by Geevies** [comment]
It doesn't drop the grab. Read the code before commenting.

**Pull request merged**